### PR TITLE
Routing setup with Express to retrieve video through request on React webpage

### PR DESCRIPTION
**Pull request by Akasora39**
Modified playback component to allow video retrieval via video id and start time (pending)
Added express
Created express route to handle video retrieval request

### CODE REVIEW

**Comment by Akasora39** [comment]
Excluding Playback.tsx and index.ts, the rest of the files can be ignored as they were just moved to a new directory.

File: src/web/server/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+console.log("Try npm run check/fix!");
+
+const longString = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer ut aliquet diam.';
+
+const trailing = 'Semicolon'
+
+            const why = 'am I tabbed?';
+
+export function doSomeStuff(withThis: string, andThat: string, andThose: string[]) {
```
Comment (chc5) [inline src/web/server/src/index.ts]: What is this

Comment (melvinheks) [reply src/web/server/src/index.ts]: Same

Comment (Akasora39) [reply src/web/server/src/index.ts]: Seems like this is something Chieh added for Typescript formatting. I'm not sure if this is still needed. @chc5

Comment (chc5) [reply src/web/server/src/index.ts]: Oh, I didn't see that the linter auto-generated this file.

**Comment by chc5** [comment]
See #27

**Pull request closed without merging**